packages/ckeditor5-source-editing/src/sourceediting.js

```diff
@@ -12,6 +12,7 @@
 import { Plugin, PendingActions } from 'ckeditor5/src/core';
 import { ButtonView } from 'ckeditor5/src/ui';
 import { createElement, ElementReplacer } from 'ckeditor5/src/utils';
+import { formatSource } from './utils/formatsource';
```

Comment:
What do you think about renaming it to `formatHtml`? This would allow moving this function someday to the `ckeditor5-utils` when there would be a need to use it elsewhere.

Reply:
Done.